alldogs.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog - Seattle's premiere dog adoption agency</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id='page-header'>
+      <a href='#start' class='skiplink'>Skip to content</a>
+      <div id='identity'>
+        <img class='logo' src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <p class='tagline'>Seattle's premiere dog adoption agency</p>
+      </div>
+      <div id='cart-bug'>
+          <img class='cart-icon' src="images/cart-icon.svg" alt="Shopping cart icon">
+          <span id='cart-total'>$1234.56</span>
+          <a href="#" class='action-button'>Checkout</a>
+      </div>
+      <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+          </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <header>
+          <h1 id='start'>Here are all of our Dogs!</h1>
+        </header>
+        <article class="">
+          <ul class='cards'>
+            <li class='card'>
+              <img src="images/murphy-card.jpg" alt="Murphy - a large brown dog with floppy ears">
+              <h2>Murphy</h2>
+              <p><strong>Cost to Adopt:</strong> $123.45</p>
+              <p>The least little bit can do so much. Everybody needs a friend.</p>
+              <a href="#" class='action-button'>Adopt</a>
+            </li>
+            <li class='card'>
```

Comment:
I like the way you lay these out, clean and simple. You MIGHT run into trouble stying the li's, and you might have to rethink how you're grouping these, but give it a shot and hopefully it'll work.